clients/fides-js/src/components/fides.css

```diff
@@ -354,7 +354,7 @@ div#fides-modal .fides-modal-button-group {
   --offset: 0.1em;
   --diameter: 1em;
 
-  display: inline-flex;
+  display: inline-flex !important;
```

Comment:
glad this worked! can you add a comment explaining why the `!important` is there, for future us?